packages/graphql/tests/tck/tck-test-files/union.test.ts

```diff
@@ -54,7 +54,108 @@ describe("Cypher Union", () => {
         });
     });
 
-    test("Read Unions", async () => {
+    test("Read Unions simple", async () => {
+        const query = gql`
+            {
+                movies {
+                    search {
+                        ... on Movie {
+                            title
+                        }
+                        ... on Genre {
+                            name
+                        }
+                    }
+                }
+            }
+        `;
+
+        const req = createJwtRequest("secret", { jwtAllowedNamesExample: ["Horror"] });
+        const result = await translateQuery(neoSchema, query, {
+            req,
+        });
+
+        expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
+            "MATCH (this:\`Movie\`)
+            CALL {
+                WITH this
+                CALL {
+                    WITH this
+                    MATCH (this)-[thisthis0:SEARCH]->(this_search:\`Genre\`)
+                    WHERE apoc.util.validatePredicate(NOT ((this_search.name IS NOT NULL AND this_search.name = $thisparam0)), \\"@neo4j/graphql/FORBIDDEN\\", [0])
+                    WITH this_search  { __resolveType: \\"Genre\\",  .name } AS this_search
+                    RETURN this_search AS this_search
+                    UNION
```

Comment:
The only suggestion I have now is if it is possible, to dedent the `UNION` by one tab. But realise this might be easier said than done!

```suggestion
                UNION
```

Reply:
Are you thinking of dedenting union or indenting the queries?
It is doable, but I think it doesn't make much sense, all the examples on Cypher docs are at the same indentation, and if you think about it, union may happen at root level:
```
MATCH(n)
RETURN n
UNION
MATCH(p)
RETURN p as n
```
So you wouldn't be able to dedent anything here :thinking: 

Another option, if you want to signify the diff of the union, is to actually indent union, although it may be a bit weird